plugins/sync_plugin.h

```diff
@@ -1,5 +1,5 @@
-#ifndef ED_HELLO_WORLD_PLUGIN_H_
-#define ED_HELLO_WORLD_PLUGIN_H_
+#ifndef ED_SYNC_PLUGIN_H_
```

Comment:
Also add the name of the guard in the `#endif` as a comment. Its a good practice.

Reply:
It isn't common in ED, will add it.

Reply:
Didn't know that. However, if you lint your code with either roslint or google style guide you'll get these warnings.